lms/templates/sysadmin_dashboard.html

```diff
@@ -163,6 +164,29 @@ <h2 class="instructor-nav">
 <hr style="width:40%" />
 %endif
 
+%if modeflag.get('task_queue'):
+  <h3>${_('Kill Queuing Tasks')}</h3><br/>
+
+  <form name="action" method="POST">
+    <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }" />
+
+    <ul class="list-input">
+      <li class="field text" style="padding-bottom: 1.2em">
```

Comment:
It would be nice if the styling was moved to SASS instead, but it looks like the CSS is just included everywhere else in this template, so it's not a big deal